.rubocop.yml

```
@@ -44,3 +44,9 @@ RSpec/ContextWording:
   Prefixes:
     - when
     - given
+
+RSpec/NestedGroups:
+  Max: 5
+
+RSpec/MultipleExpectations:
+  Max: 10
```

Should we also exclude the system test here? as we tend to have more `expect` per test on that part?

```suggestion
  Max: 10
  Exclude:
    - "spec/systems/**/*"
```